Sofa/framework/Simulation/Core/src/sofa/simulation/WorkerThread.cpp

```diff
@@ -69,6 +73,13 @@ std::thread *WorkerThread::create_and_attach(DefaultTaskScheduler *const &taskSc
 
 void WorkerThread::run(void)
 {
+#ifdef WIN32
+    const std::wstring widestr = std::wstring(m_name.begin(), m_name.end());
+    HRESULT r = SetThreadDescription(
+        GetCurrentThread(),
+        widestr.c_str()
+        );
+#endif
```

Comment:
not needed or not possible on other OS ?

Reply:
Naming the threads is useful (only?) for debugging. This method is limited to Visual Studio.
I don't know if the same feature exists for other debuggers on other platforms.

Reply:
Seems to be possible with pthread at least: https://stackoverflow.com/questions/2369738/how-to-set-the-name-of-a-thread-in-linux-pthreads
for a future PR I suppose